src/components/sync-sites-modal-selector.tsx

```diff
@@ -129,8 +129,9 @@ const getSortedSites = ( sites: SyncSite[] ) => {
 	const order: Record< SyncSite[ 'syncSupport' ], number > = {
 		syncable: 1,
 		'already-connected': 2,
-		'needs-transfer': 3,
-		unsupported: 4,
+		deleted: 3,
```

Comment:
Could we move `deleted` below `needs-transfer`? The reason is that the latter still have a chance to change their status, but the former can't.

Reply:
Just to share my thoughts - I was thinking about it and it was intentional, exactly there - we render only "deleted && connected", so we will render literally maximum 1 deleted site, and we need to render it at the top, to make it noticeable for user.

But I am ok to move it below `need-transfer` if my argument doesn't sound rational.

Reply:
It makes sense, thanks for providing more details.